### DISCUSSION TITLE
PD2 Compatible?

### DISCUSSION THREAD

**Discussion by Faramirdx** [isid0re/SoloLeveling#41] [General]
Hey, i just was wondering if this script will also work in PD2.
Greetings :)

**Comment by isid0re** [comment]
I haven't ran my script on pd2.  BUT, I've been told that it does work. And I believe it because there have been some unscrupulous people reselling my work as their own for pd2.

**Reply by Faramirdx** [reply]
Well reselling it is really poor... 
I've tested it a bit with the Sorc and yes in kinda works. But since the skills have changed shes always trying to cast Enchand Cold (new Skill), I believe shes trying to cast Frozen Armor (?!). Havent found the build .js where i could change that. Would be great if you could help me out there.

**Reply by isid0re** [reply]
All build files are located in the sololeveling/buildfiles folder 

If you want to change the build around you will need to edit the skill settings there.

**Reply by Faramirdx** [reply]
Alright, thx will have a look there
